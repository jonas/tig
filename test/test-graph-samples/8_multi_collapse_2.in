commit A Z B C D E F
    Commit A - merge Z, B, C, D, E, and F
commit C B
    Commit C - after B
commit B H
    Commit B - after H
commit F G
    Commit F - after G
commit H D
    Commit H - after D
commit D E
    Commit D - after E
commit E G
    Commit E - after G
commit G Z
    Commit G - after Z
commit Z
    Commit Z
