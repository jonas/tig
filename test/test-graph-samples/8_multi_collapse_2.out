●─┬─┬─┬─┬─╮ Commit A - merge Z, B, C, D, E, and F
│ │ ● │ │ │ Commit C - after B
│ ●─╯ │ │ │ Commit B - after H
│ │ ╭─╯ │ ● Commit F - after G
│ ● │ ╭─╯ │ Commit H - after D
│ ●─╯ │ ╭─╯ Commit D - after E
│ ●───╯ │ Commit E - after G
│ ●─────╯ Commit G - after Z
◎─╯ Commit Z
